Makefile

```diff
@@ -0,0 +1,8 @@
+build:
+	npm --version
+	node --version
+	npm install -g yarn@1.5.1
+	yarn --version
+	yarn
+	yarn lint
+	yarn build
```

Comment:
The website needs yarn?

Reply:
Yeah, doesn't require yarn but yarn is faster + less risk of missing packages. It should work with both though, so I have no strong feelings on this. Probably we should move back to npm if it has matured lately, for consistency.